### PR TITLE
Support openHardwareMonitor lib

### PR DESCRIPTION
**Pull request by ibrahemDev**

### CODE REVIEW

**Comment by w-e-w** [comment]
I take it that it's still not ready because you said it as a draft

File: scripts/install.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import launch
+import os
+from modules import shared
```
Comment (w-e-w) [inline scripts/install.py]: importing `share` in `installer` is a bad idea
`share` includes importing of `gradio`
which may dramatically slow load time

**Comment by ibrahemDev** [comment]
> I take it that it's still not ready because you said it as a draft

yup I'am trying to hide "gpu_temps_sleep_gpu_name" when "gpu_temps_sleep_temperature_src " changed without click "Apply settings"

File: scripts/install.py (hunk 1 above)
Comment (ibrahemDev) [reply scripts/install.py]: :|  
ok can I store "gpu_temps_sleep_temperature_src" value on json file ? 
I want only install  when select "NVIDIA & AMD - openHardwareMonitor"

**Comment by w-e-w** [comment]
is there a reason why you're creating a separate setting file?

if the reason is to use it for install.py then with the standard method then yes I can see the reason
but since you're not, I don't understand

I might make some modifications

**Comment by ibrahemDev** [comment]
Yes the reason for use it for install.py 
> importing share in installer is a bad idea
> share includes importing of gradio
> which may dramatically slow load time


> I might make some modifications

I added you as collaborator to my fork so I think you have access to make changes

**Comment by w-e-w** [comment]
oh you actually don't need to add me as collaborator because you already give me access by makeing a PR and haveing the allowing maintain it to make changes

image

I'll make changes later
I'm a bit busy at the moment not sure when I will get it

**Comment by ibrahemDev** [comment]
Great, in fact, I'm not an expert enough in github, but I still learning it :D

> I'll make changes later

feel free

**Comment by w-e-w** [comment]
>  I still learning

we all are

**Comment by w-e-w** [comment]
@ibrahemDev
I reworked it can you test on your machine with on new install
it should be possible to just enable OpenHardwareMonitor without a restart
if you can confirm that this works then I will merge it

**Comment by ibrahemDev** [comment]
yup it is work perfectly without restart
I removed all lib ("OpenHardwareMonitorLib.dll" and "pythonnet" and "clr") before try

**Pull request merged**